CS342MemberViews/Sources/JenMoore/JenMoore.swift

```diff
@@ -0,0 +1,53 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+public struct JenMoore: View {
+    @Binding var navigationPath: NavigationPath
+    @State var accentColor = Color(red: 134 / 255, green: 89 / 255, blue: 77 / 255)
```

Comment:
If you remove the `@Bindings` from the other views, you can also make this a constant and remove the `@State` property wrapper:
```suggestion
    let accentColor = Color(red: 134 / 255, green: 89 / 255, blue: 77 / 255)
```